src/tidy.zig

```diff
@@ -213,6 +215,50 @@ test "tidy naughty list" {
     }
 }
 
+test "tidy no large blobs" {
+    const allocator = std.testing.allocator;
+    const shell = try Shell.create(allocator);
+    defer shell.destroy();
+
+    // Run `git rev-list | git cat-file` to find large blobs. This is better than looking at the
+    // files in the working tree, because it catches the cases where a large file is "removed" by
+    // reverting the commit.
+    //
+    // Zig's std doesn't provide a cross platform abstraction for piping two commands together, so
+    // we begrudgingly pass the data through this intermediary process.
+    const MiB = 1024 * 1024;
+    const rev_list = try shell.exec_stdout_options(
+        .{ .max_output_bytes = 50 * MiB },
+        "git rev-list --objects HEAD",
+        .{},
+    );
+    const objects = try shell.exec_stdout_options(
+        .{ .max_output_bytes = 50 * MiB, .stdin_slice = rev_list },
+        "git cat-file --batch-check={format}",
+        .{ .format = "%(objecttype) %(objectsize) %(rest)" },
+    );
+
+    var has_large_blobs = false;
+    var lines = std.mem.split(u8, objects, "\n");
+    while (lines.next()) |line| {
+        // Parsing lines like
+        //     blob 1032 client/package.json
+        var blob = stdx.cut_prefix(line, "blob ") orelse continue;
+
+        var cut = stdx.cut(blob, " ").?;
+        const size = try std.fmt.parseInt(u64, cut.prefix, 10);
+        const path = cut.suffix;
+
+        if (std.mem.eql(u8, path, "src/vsr/replica.zig")) continue; // :-)
```

Comment:
I'm not sure I quite follow why replica.zig and package-lock.json are here - aren't they both less than 4MB?

Reply:
`@divExact(MiB, 4)` is a quarter of, not four megs

Reply:
Ahhh. My brain just did not want to parse that!